pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -246,17 +253,14 @@ def on_advance_end(self) -> None:
             self._run_validation()
             self.trainer.training = True
 
-        # -----------------------------------------
-        # SAVE LOGGERS (ie: Tensorboard, etc...)
-        # -----------------------------------------
-        self._save_loggers_on_train_batch_end()
```

Comment:
I don't understand why we change the order if loggers depend on the fake counter. I would not expect the behavior to change for loggers. For them, what once was named global step is now called `_batches_that_stepped`.

Reply:
This change is not strictly necessary, but I found it more intuitive to put it alongside the increase, also having it after avoids the `+ 1` inside `_save_loggers_on_train_batch_end`